app/lib/methods/getSlashCommands.js

```diff
@@ -0,0 +1,32 @@
+import { InteractionManager } from 'react-native';
+
+import database from '../realm';
+import log from '../../utils/log';
+
+export default async function() {
+	try {
+		// RC 0.70.0
+		const result = await this.sdk.get('commands.list');
+
+		if (!result.success) {
+			log('getSlashCommand fetch', result);
```

Comment:
`return log('err_get_slash_command_fetch', result)`

Reply:
If you don't add return, the rest of the code will run even after receiving an error